Src/WitsmlExplorer.Api/Workers/Modify/ModifyUtils.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+using WitsmlExplorer.Api.Models.Measure;
+
+namespace WitsmlExplorer.Api.Workers.Modify
+{
+    public class ModifyUtils
```

Comment:
`ModifyUtils` can be made static.

```suggestion
    public static class ModifyUtils
```